iotilecore/test/test_reports/test_utc_assigner.py

```diff
@@ -63,6 +64,8 @@ def compare_fixed_report(fixed, ref_filename):
     assert len(fixed.visible_readings) == len(ref_lines)
 
     for reading, ref_string in zip(fixed.visible_readings, ref_lines):
+        if str(reading) != ref_string:
+            print("FAIL: {} != {}".format(str(reading),ref_string))
```

Comment:
Is this check redundant with the `assert`? The assert below should take care of this check now.